tools/hf_transformers/convert_checkpoint.py

```diff
@@ -0,0 +1,323 @@
+import argparse
```

Comment:
Is this an exact copy from transformers? Same for push_checkpoints.py

Reply:
`convert_checkpoints.py` is largely copied form transformers, but includes support for the MQA models.
`push_checkpoints.py` is new code.
This folder is named hf_transformers because these are tools to facilitate usage of Megatron models in transformers